cmplibrary/src/main/java/com/sourcepoint/cmplibrary/util/ViewsManager.kt

```diff
@@ -155,7 +150,6 @@ private class ViewsManagerImpl(
             check {
                 ConsentWebView(
                     context = it,
-                    connectionManager = connectionManager,
```

Comment:
no need this change

Reply:
this change is needed, if we delete `ConnectionManager`

Reply:
the same here